common/reviews/api/webpack5-localization-plugin.api.md

```diff
@@ -121,30 +126,40 @@ export interface IPseudolocalesOptions {
 export type IResolvedMissingTranslations = ReadonlyMap<string, ILocaleFileData>;
 
 // @public (undocumented)
-export interface _IStringPlaceholder {
+interface IStringPlaceholder extends IValuePlaceholderBase {
     locFilePath: string;
     stringName: string;
-    suffix: string;
     translations: ReadonlyMap<string, ReadonlyMap<string, string>>;
-    value: string;
 }
+export { IStringPlaceholder }
+export { IStringPlaceholder as _IStringPlaceholder }
```

Comment:
Weird to export the same interface twice from the same file

Reply:
This is to avoid a breaking API change.